src/CapabilityService.WebApi/Properties/launchSettings.json

```diff
@@ -32,7 +32,8 @@
 			"CAPABILITY_SERVICE_KAFKA_TOPIC_CAPABILITY": "build.selfservice.events.capabilities",
 			"CAPABILITY_SERVICE_KAFKA_TOPIC_TOPICS": "build.selfservice.events.topics",
 			"CAPABILITY_SERVICE_KAFKA_ENABLE_AUTO_COMMIT": "false",
-			"KAFKAJANITOR_API_ENDPOINT": "http://localhost:5000"
+			"KAFKAJANITOR_API_ENDPOINT": "http://localhost:5000",
+			"CAPABILITYSERVICE__CORS__ORIGIN": "http://localhost:3000"
```

Comment:
Shouldnt we follow the same naming scheme as the other envs? == CAPABILITY_SERVICER_CORS_ORIGIN?

Reply:
Added!